Sources/AblyAssetTrackingPublisher/DefaultPublisherBuilder.swift

```diff
@@ -22,7 +23,9 @@ class DefaultPublisherBuilder: PublisherBuilder {
                  routingProfile: RoutingProfile?,
                  delegate: PublisherDelegate?,
                  resolutionPolicyFactory: ResolutionPolicyFactory?,
-                 areRawLocationsEnabled: Bool?) {
+                 areRawLocationsEnabled: Bool?,
+                 isSendResolutionEnabled: Bool = false
```

Comment:
Like above, I think that sending resolution should be enabled by default :wink:

Reply:
changed a288514997ea39ae67ca98329302cc66012e4595